### ISSUE TITLE
CLI installer falis

### ISSUE THREAD

**Issue by cb9** [Antergos/Cnchi#216]
I have tried to install using CLI installer last year september (Spetember 03,2013) and June 02,2014. I tried to install Openbox desktop. It starts parallel download of the packages and after long time it give error saying "Insallation failed". After pressing Enter on OK button, it says "Refreshing Package Database" The it says "Installing Packages please wait ... " And again says "Installation failed". And agin it repeats "Refreshing Package Database" and again says "Installation failed" This repeats. I had to finally use Graphic installer. 
I think the problem is when it tries to do parallel download of packages, may be some of the packages are not downloaded properly, this is why installation fails. I think if instead of downloading packages in parallel if it downloads serially, like graphic installer, this error can be fixed. This is just my guess, but I do not know exact reason behind it.

**Comment by karasu** [comment]
I'm afraid we're no longer maintaining the cli installer. If you have problems with the graphical one please open a new issue.
Thanks!
